src/Internal/DependencyManagement/AbstractServiceProvider.php

```diff
@@ -78,21 +76,20 @@ protected function add_with_auto_arguments( string $class_name, $concrete = null
 	private function reflect_class_or_callable( string $class_name, $concrete ) {
```

Comment:
The method comment still has a couple of mentions to the constructor, those should be adjusted.